src/App.tsx

```diff
@@ -17,18 +50,32 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                setFilter={setCurrentFilter}
+                setSearch={setCurrentSearch}
```

Comment:
The naming convention for event handler functions and props suggests using 'on' prefix for event handlers and 'handle' prefix for internal methods. Consider renaming 'setFilter' and 'setSearch' to follow this convention.